utils/navigator_tools/navigator_tools/general_helpers.py

```diff
@@ -3,6 +3,10 @@
 def print_t(_str, time=None):
     # Needs a rospy init for ros time
     if time is None:
-        time = rospy.Time.now().to_sec()
-
-    print "\033[1m{}:\033[0m {}".format(time, _str)
\ No newline at end of file
+        try:
+            time = rospy.Time.now().to_sec()
+            print "\033[1m{}:\033[0m {}".format(time, _str)
+        except rospy.exceptions.ROSInitException:
+            print _str
```

Comment:
I can add printing system time, but I think it would be confusing with a different time "position" and time scale.